package.json

```
@@ -39,6 +39,7 @@
     "@sentry/cli-win32-x64": "2.23.1"
   },
   "scripts": {
+    "postinstall": "node ./scripts/install.js",
```

Since we have `usr/bin/env node` in the first line of the script, we don't need `node` here:

```suggestion
    "postinstall": "./scripts/install.js",
```

---

I am not sure if the shebang will work on windows. Both SWC and esbuild have their postinstall scripts defined like this and it doesn't seem it's like that by accident so I will keep it.

---

Actually, see https://github.com/getsentry/sentry-cli/pull/1151

---

@lforst It break in Amplify environment
```
npm ERR! path /root/.nvm/versions/node/v16.20.2/lib/node_modules/@sentry/cli
npm ERR! command failed
npm ERR! command sh -c -- node ./scripts/install.js
npm ERR! sh: node: command not found
```